api/v1alpha1/dspipeline_types.go

```diff
@@ -158,18 +182,22 @@ type ExternalDB struct {
 type ObjectStorage struct {
 	*Minio           `json:"minio,omitempty"`
 	*ExternalStorage `json:"externalStorage,omitempty"`
+	// Default: false
 	// +kubebuilder:default:=false
 	// +kubebuilder:validation:Optional
 	DisableHealthCheck bool `json:"disableHealthCheck"`
 }
 
 type Minio struct {
+	// Default: true
 	// +kubebuilder:default:=true
 	// +kubebuilder:validation:Optional
 	Deploy bool `json:"deploy"`
+	// Default: mlpipeline
 	// +kubebuilder:default:=mlpipeline
 	Bucket              string `json:"bucket,omitempty"`
 	*S3CredentialSecret `json:"s3CredentialsSecret,omitempty"`
```

Comment:
Credentials for the S3 user (e.g. IAM user cred stored in a k8s secret.).
Note that the S3 user should have the permissions to create a bucket if the provided bucket does not exist.